content/guides/asset-management.md

```diff
@@ -4,163 +4,522 @@ sort: 3
+5c999da72346a995e7e2718865d019c8.png  11.3 kB          [emitted]
+                           bundle.js   561 kB       0  [emitted]  [big]  main
+   [0] ./src/icon.png 82 bytes {0} [built]
+   [1] ./~/lodash/lodash.js 540 kB {0} [built]
+   [2] ./src/style.css 1 kB {0} [built]
+   [3] ./~/css-loader!./src/style.css 242 bytes {0} [built]
+   [4] ./~/css-loader/lib/css-base.js 2.26 kB {0} [built]
+   [5] ./~/style-loader/lib/addStyles.js 8.7 kB {0} [built]
+   [6] ./~/style-loader/lib/urls.js 3.01 kB {0} [built]
+   [7] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [8] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [9] ./src/index.js 503 bytes {0} [built]
+```
+
+If all went well, you should now see your icon as a repeating background, as well as an `img` element behind our `Hello webpack` text. If you inspect this element, you'll see that actual filename has changed to something like `5c999da72346a995e7e2718865d019c8.png`. This means webpack found our file in the `src` folder and processed it!
+
+T> A logical next step from here is minifying and optimizing your images. Check out the [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader) and [url-loader](/loaders/url-loader) for more on how you can enhance your image loading process.
 
 
 ## Loading Fonts
 
-So what about other assets like fonts? The file and url loaders will take any file you load through them and output it to your build directory. This means we can use them for any kind of file, including fonts:
+So what about other assets like fonts? The file and url loaders will take any file you load through them and output it to your build directory. This means we can use them for any kind of file, including fonts. Let's adjust our `webpack.config.js` file to instruct it to handle font files:
 
 __webpack.config.js__
 
-``` js
+``` diff
+var path = require('path');
+
 module.exports = {
-  //...
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  },
   module: {
     rules: [
       {
-        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader'
+        ]
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
         use: [
           'file-loader'
         ]
-      }
+      },
++       {
++         test: /\.(woff|woff2|eot|ttf|otf)$/,
++         use: [
++           'file-loader'
++         ]
++       }
     ]
-  },
-  //...
-}
+  }
+};
+
 ```
 
-With this in place you can define a font like so...
+Add some font files to your project:
 
-``` css
-@font-face {
-  font-family: 'MyFont';
-  src:  url('./font.woff2') format('woff2'),
-        url('./font.woff') format('woff');
-  font-weight: 600;
-  font-style: normal;
+__project__
+
+
+``` diff
+ |- package.json
+ |- webpack.config.js
+ |- dist
+   |- bundle.js
+   |- index.html
+ |- /src
++   |- MyFont.woff
++   |- MyFont.woff2
```

Comment:
This is somewhat nit-picky but I would suggest sticking with lower cased file names so either `font.woff`/`font.woff2` or `my-font.woff`/`my-font.woff2`.